lib/scheme-refresh-token.js

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const AppError = require('./error')
+const joi = require('joi')
+const validators = require('./routes/validators')
+const { BEARER_AUTH_REGEX } = require('./routes/validators')
+const ScopeSet = require('fxa-shared').oauth.scopes
+
+const ALLOWED_REFRESH_TOKEN_SCHEME_SCOPES = ScopeSet.fromArray(['https://identity.mozilla.com/apps/oldsync'])
```

Comment:
I think this would really benefit from a comment about why this scope is the only one allowed (because this API is only currently used by things connected to sync, and we're at a transitionary stage of its evolution into something more generic, so it's here as a safety mechanism).

Reply:
👍